dinky-admin/src/main/java/org/dinky/controller/BaseController.java

```diff
@@ -0,0 +1,22 @@
+package org.dinky.controller;
+
+import org.dinky.common.result.Result;
+import org.dinky.constant.BaseConstant;
+import org.dinky.enums.Status;
+
+import java.util.Map;
+
+public abstract class BaseController {
+    /**
+     * this method returns a successful result with the status
+     *
+     * @param status
+     * @return result code
+     */
+    public static Result<Void> getResultFromStatus(Status status){
```

Comment:
actually, `Result` as a tool class, already contains a lot of common method, to dinky. you should consider use and improve it, the seemed no requirement to add a `BaseController`

Reply:
What I think is that Result should be used as the return value, and it should be done well to directly return the result. If all other operations are in Result, will it violate the single principle of the class?

Reply:
I'm not sure what operation you want to do, but if it's about the result format or converter, etc, it's ok, they are `Result`'s duty, if it's about some others, you could consider to extends `Result` class, or add a tool class.

Reply:
What I want to do is that when we finally obtain the result, we may need to process the Result, just like the method I wrote above, which has their own status to obtain the final result. If it is written in the Result, it feels too coupled. If it is placed in the tool class, it feels unnecessary. My idea is to put it in their parent class, which can have universal methods, I don't rely too much on other tool classes either. I think so

Reply:
Hello, according to your request, I want to follow the big shots and make some simple things in the early stages. Do you think it's better to write this section directly to Result, or is it better to write it as a tool class, or is that all?

Reply:
Okay, thank you. I have made the necessary modifications. Is that the effect you want?